example/autoencoder/solver.py

```diff
@@ -19,9 +19,8 @@
 from __future__ import print_function
 
 import logging
-
-import mxnet as mx
 import numpy as np
+import mxnet as mx
```

Comment:
Why mxnet import should be moved down?

Reply:
Solve pylint error message such as [C0411 ] standard import "import numpy as np" should be placed before "import mxnet as mx"